tools/Equinox.Tool/Program.fs

```diff
@@ -396,16 +396,16 @@ module Dump =
             let source = if not doE && not (List.isEmpty unfolds) then Seq.ofList unfolds else Seq.append events unfolds
             let mutable prevTs = None
             for x in source |> Seq.filter (fun e -> (e.IsUnfold && doU) || (not e.IsUnfold && doE)) do
-                let ty = if x.IsUnfold then "Unfold" else "Event"
+                let ty,render = if x.IsUnfold then "U", render Newtonsoft.Json.Formatting.Indented else "E", render fo
                 let interval =
                     match prevTs with Some p when not x.IsUnfold -> Some (x.Timestamp - p) | _ -> None
                     |> function
                     | None -> if doT then "n/a" else "0"
                     | Some (i : TimeSpan) when not doT -> i.ToString()
                     | Some (i : TimeSpan) when i.TotalDays >= 1. -> i.ToString "d\dhh\hmm\m"
                     | Some i when i.TotalHours >= 1. -> i.ToString "h\hmm\mss\s"
-                    | Some i when i.TotalMinutes >= 1. -> i.ToString "m\mss\s"
-                    | Some i -> i.ToString("s\.ff\s")
+                    | Some i when i.TotalMinutes >= 1. -> i.ToString "m\mss\.f\s"
```

Comment:
Generally these timestamps are represented as "m/mss/" with slashes going the other way. Is it the opposite way outside of the US?

Reply:
escaping the char (this is a stray commit that shouldnt be here!)